src/autopr/main.d

```diff
@@ -205,7 +209,7 @@ void main ( string[] args )
         // Merge the new data in the existing result
         mergeJson(qresult, result);
     }
-    while (more_pages);
+    while (more_pages || !node_chunks.empty);
```

Comment:
`&&`?

Reply:
No, we want all chunks, even if they are no more pages for the other type of data we fetch